app/views/steps/_form_assets.html.erb

```diff
@@ -12,7 +12,7 @@
   </div>
   <div class="panel-body">
     <% if ff.object.file.exists? %>
-      <% if @protocol.in_module? && can_view_or_download_step_assets(@protocol) || @protocol.in_repository? && can_read_protocol_in_repository?(@protocol) %>
+      <% if can_read_protocol_in_module?(@protocol) || can_read_protocol_in_repository?(@protocol) %>
```

Comment:
where we declare this can_read_protocol_in_repository?

Reply:
In `/app/permissions/team.rb`.

Reply:
:+1: